tests/dummy/app/controllers/index.js

```diff
@@ -1,4 +1,5 @@
 import Controller from '@ember/controller';
+import { A } from '@ember/array';
```

Comment:
Why do we use it?

Reply:
Because of this https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/avoid-leaking-state-in-ember-objects.md. Do you think ignoring it in this case would make better sense?

Reply:
So as you can see in the example, instead of Ember.A we should just initialize the array in the init:
```js
export default Foo.extend({
  items: null,

  init() {
    this._super(...arguments);
    this.items = [];
  },
}
```